routes/auth.js

```diff
@@ -11,72 +11,74 @@ router.get('/', (req, res) => {
 
 
 router.get('/users', passport.authenticate('jwt', { session: false }), async (req, res) => {
-  try{
+  try {
     const users = await User.find()
     res.json(users)
   } catch (e) {
-    res.status(500).json({'message': e.message});
+    res.status(500).json({ 'message': e.message });
   }
 })
 
 router.post('/register', async (req, res) => {
   try {
-    let userExists = await User.find({email: req.body.email});
-    if(Array.isArray(userExists) && userExists.length){
+    let userExists = await User.find({ email: req.body.email });
+    if (Array.isArray(userExists) && userExists.length) {
       res.redirect('/login'); // TODO: notify user existence
-    } else{
-        const user = new User({
-          name: req.body.name,
-          password: await bcrypt.hash(req.body.password, 10),
-          email: req.body.email,
-          phone: req.body.phone,
-          isAdmin: JSON.parse(req.body.isAdmin) // Convert admin to boolean
-        });
+    } else {
+      const user = new User({
+        firstname: req.body.firstname,
+        lastname: req.body.lastname,
+        password: await bcrypt.hash(req.body.password, 10),
+        email: req.body.email,
+        phone: req.body.phone,
+        isAdmin: JSON.parse(req.body.isAdmin) // Convert admin to boolean
```

Comment:
Should we remove this field now, or hold off till later?

Reply:
I guess we can keep it as long as our front end is passing isAdmin field in request body or else it would error out.
Also it's more convinient for us to create an admin account.